New syntax for `dotnet new` (#28857)
dotnet/docs: includes/dotnet-new-7-0-syntax.md

---
author: vshubina
ms.author: vshubina
ms.date: 30/03/2022
ms.topic: include
---

## .NET SDK 7.0.100 `dotnet new` syntax

Starting .NET SDK 7.0.100  Preview 2 `dotnet new` syntax has changed:

> - `--list`, `--search`, `--install` `--uninstall` options became `list`, `search`, `install` and `uninstall` subcommand.

Reviewer: ```suggestion
- The `--list`, `--search`, `--install`, and `--uninstall` options became `list`, `search`, `install` and `uninstall` subcommands.
```

- `--update-apply` option became `update` subcommand.
- to use `--update-check`, use `update` subcommand with `--check` option.

> Other options available before are still available to use with respective subcommands.

Reviewer 2: ```suggestion
Other options available before are still available to use with their respective subcommands.
```

> Other options available before are still available to use with respective subcommands.

Reviewer: ```suggestion
Other options that were available before are still available to use with their respective subcommands.
```

> There is a separate help for each subcommand avilable via `-h` or `--help` option: `dotnet new <subcommand> --help`, that lists all supported options for the subcommand.

Reviewer: ```suggestion
Separate help for each subcommand is available via the `-h` or `--help` option: `dotnet new <subcommand> --help` lists all supported options for the subcommand.
```

> Additionally, the tab completion is now available for `dotnet new`.

Reviewer: ```suggestion
Additionally, tab completion is now available for `dotnet new`.
```

> To activate tab completion for .NET SDK, follow [the guideline](../docs/core/tools/enable-tab-autocomplete.md).

Reviewer: ```suggestion
To activate tab completion for the .NET SDK, see [Enable tab completion](../docs/core/tools/enable-tab-autocomplete.md).
```
